internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -153,3 +158,32 @@ CertificateReplicator:
     CertificateArn: !Ref HTTPSCert
 {{- end}}
 {{- end}}
+
+{{- if .CDNConfig.Static}}
+{{- if .CDNConfig.Static.StaticSiteBucket}}
+{{logicalIDSafe (bucketNameFromURL .CDNConfig.Static.StaticSiteBucket)}}BucketPolicyForCloudFront:
+  Metadata:
+    'aws:copilot:description': 'A bucket policy to grant CloudFront read access to the Static Site bucket "{{bucketNameFromURL .CDNConfig.Static.StaticSiteBucket}}"'
+  Type: AWS::S3::BucketPolicy
+  DeletionPolicy: Retain
+  Condition: CreateALB
+  Properties:
+    Bucket: {{bucketNameFromURL .CDNConfig.Static.StaticSiteBucket}}
```

Comment:
will this cause the static site to fail to delete? because the policy, owned by this stack, hasn't been deleted?

or, will the env stack fail to delete/update after the static site is deleted, because this resource has a drift (if it got deleted along with the bucket)?

Reply:
I've tested out and it's not tracked by drift detection and won't cause the workload/env stack deletion failure